.pre-commit-config.yaml

```
@@ -1,23 +1,24 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 21.7b0
+    rev: 22.8.0
     hooks:
     -   id: black
         args: [--safe, --quiet]
 -   repo: https://github.com/asottile/blacken-docs
-    rev: v1.10.0
+    rev: v1.12.1
     hooks:
     -   id: blacken-docs
-        additional_dependencies: [black==21.7b0]
+        additional_dependencies: [black==22.6.0]
```

To match the one above:
```suggestion
        additional_dependencies: [black==22.8.0]
```